docs/howto/build-run.md

```diff
@@ -499,3 +505,16 @@ Optionally, reset iOS simulator:
 ```
 iOS Menu > Simulator > Reset Content and Settings…
 ```
+
+If you get other build failures on iOS and you haven't changed
+anything in the `ios` folder yourself, there might residue from a
+previous build interfering in this one. So, try cleaning the build
+folder. A different folder is used for command-line builds and builds
+through Xcode.
+- If building from the command line with `react-native run-ios`, run
+  `rm -rf ios/build`.
```

Comment:
Huh, I didn't know that! I guess I haven't used `react-native run-ios` in a long time :-) -- for whatever reason, I always use Xcode for running debug versions on iOS, even though I always use `react-native run-android` for Android.

Reply:
Yep!

My heart skipped a beat when I realized, several commits on `master` after CocoaPods landed, that I could consistently build with Xcode and consistently NOT build with `react-native run-ios`. This fact made me doubt whether I had indeed done the full testing necessary (including building from the command line) before CocoaPods got merged.

Happily, I discovered that two different build folders are used, and, on clearing them both and finding that both builds worked fine afterward, I much more confidently remembered that I'd done all the required testing. 😝